build/dockerfiles/deb.Dockerfile

```diff
@@ -18,6 +18,7 @@ ARG KONG_ARTIFACT_PATH=
 COPY ${KONG_ARTIFACT_PATH}${KONG_ARTIFACT} /tmp/kong.deb
 
 RUN apt-get update \
+    && apt-get -y upgrade \
```

Comment:
Maybe also add `apt autoremove` afterwards to remove unused dependencies?

Reply:
Make sense. Updated.